fem/lininteg_domain.cpp

```diff
@@ -25,7 +25,7 @@ void DLFEvalAssemble2D(const int vdim, const int ne, const int d, const int q,
    const auto F = coeff.Read();
    const auto M = Reshape(markers, ne);
    const auto B = Reshape(b, q, d);
-   const auto J = Reshape(j, q, q, 2,2, ne);
+   const auto J = Reshape(j, q, q, ne);
```

Comment:
Could we change `J` to `D` maybe?

Reply:
How about `DETJ`? `D` is already used for `constexpr int D = T_D1D ? T_D1D : MAX_D1D;`

Reply:
Done in 106964e